focuspoints.lrdevplugin/DefaultDelegates.lua

```diff
@@ -24,6 +24,7 @@ require "Utils"
 
 DefaultDelegates = {}
 DefaultDelegates.focusPointsMap = nil
+DefaultDelegates.focusPointDimen = nil
```

Comment:
I merged this into a branch "center_it" to run it. the property is null at run time. I can take care of it sometime or feel free or you get time.